documentation/SystemDesign/Design.tex

```diff
@@ -128,7 +128,18 @@ \subsubsection{Monitored Variables}
 \subsubsection{Controlled Variables}
 
 \subsection{Subsystems}
-The system will be broken down into the subsystems with the following purposes: general managing of subsystems, computer vision to detect returns, shot recommendation, modelling the shot's trajectory, optimizing the distance travelled by the shooting mechanism, storing the data, taking input and output from the user and shooting the ball toward the user. The diagram for this breakdown can be found in Figure \ref{fig:sub}.
+The system will be broken down into subsystems with the following purposes: 
+\begin{itemize}
+\item \textbf{SmartServe}: general management of subsystems
+\item \textbf{Computer Vision}:  detection of returns
+\item \textbf{Shooting Mechanism}:  shooting the ball toward the user
+\item \textbf{Shot Recommendation}: provides best shot
+\item \textbf{Data Storage}: storing the data
+\item \textbf{Shooting Model}: modelling the shot's trajectory
+\item \textbf{Shot Optimizer}: optimizing the distance travelled by the shooting mechanism
+\item \textbf{User Interface}: taking input from the user and showing output
+\end{itemize}
+The diagram for this breakdown can be found in Figure \ref{fig:sub}.
```

Comment:
We should add a TODO to add clever references (i.e. the ones where "Figure 1" is all clickable)